test-assets/SC6-1-2-iframe-unique-name-doc3.html

```diff
@@ -0,0 +1,6 @@
+Page with nested frame.
+<iframe 
+	src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html"
+	title="wait-i-have-the-same-name"
```

Comment:
How are you testing that for nested frames there is no issue? Also, should we test that within iframes this still holds?